docs/abscomp.rst

```diff
@@ -0,0 +1,72 @@
+.. highlight:: rest
+
+******************
+AbsComponent Class
+******************
+
+.. index:: AbsComponent
+
+Overview
+========
+
+This Class is designed to organize and analyze a set of
+absorption lines (:ref:`AbsLine Class`).
+
+By definition, an AbsComponent is a unique collection of
+absorption lines specified by:
+
+=============== ========   ============== ============================================
+Property        Variable   Type           Description
+=============== ========   ============== ============================================
+RA, DEC         radec      tuple          RA,DEC in deg
```

Comment:
I think that is asking the user a bit too much.  

But, I'm fine if we allow it to be an astropy.coodrinate
Am modifying it that way.